libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -287,30 +258,52 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
 - (void)clearAllAccountState;
 
 /** Invoke this method to apply the specified credentials to the
- current user. If no user exists, a new one is created.
+ a user whose credentials match. If no user exists, a new one is created.
  This will post user update notification.
  @param credentials The credentials to apply
  */
-- (void)applyCredentials:(SFOAuthCredentials*)credentials;
+- (SFUserAccount *) applyCredentials:(SFOAuthCredentials*)credentials;
 
 /** Invoke this method to apply the specified id data to the
  current user. This will post user update notification.
  @param idData The ID data to apply
  */
 - (void)applyIdData:(SFIdentityData *)idData;
```

Comment:
Given our commitment to act per user, should we remove these "current user" convenience methods?  They seem like an anti-pattern with our updated implementation.